Core/clim-core/table-formatting.lisp

```diff
@@ -373,60 +435,69 @@ skips intervening non-table output record structures."))
                                             standard-sequence-output-record)
   ((x-spacing :initarg :x-spacing)
    (y-spacing :initarg :y-spacing)
-   (initial-spacing :initarg :initial-spacing)
-   (row-wise :initarg :row-wise)
-   (n-rows :initarg :n-rows)
    (n-columns :initarg :n-columns)
+   (n-rows :initarg :n-rows)
    (max-width :initarg :max-width)
-   (max-height :initarg :max-height)))
+   (max-height :initarg :max-height)
+   (initial-spacing :initarg :initial-spacing)
+   (row-wise :initarg :row-wise)))
 
 (defmethod map-over-item-cells (function
                                 (item-list-record standard-item-list-output-record))
   (map-over-block-cells function item-list-record))
 
-(defun invoke-format-item-list (stream continuation
-                                &key x-spacing y-spacing n-columns n-rows
-                                  max-width max-height
-                                  initial-spacing (row-wise t) (move-cursor t)
-                                  (record-type
-                                   'standard-item-list-output-record))
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (alexandria:define-constant +standard-item-list-output-record-initargs+
+      '(x-spacing y-spacing n-columns n-rows max-width max-height
+        initial-spacing row-wise stream-width stream-height)
+    :test #'equal))
```

Comment:
these are item-list-output-record initiargs, which is a protocol class - rename a constant. also please use keywords, not internal symbols.